config/locales/en.yml

```diff
@@ -9276,10 +9276,28 @@ en:
         trending_in_place_desc_html: Most-observed taxa in the last month in <a href="%{url}">%{place}</a>
         wanted_desc: Species in this taxon that have not been observed yet.
     taxon_changes:
+      # warning when changing taxon change status from withdrawn back to draft
+      are_you_sure_you_want_to_restore_this_taxon_change: Are you sure you want to restore this taxon change?
+      # warning when changing taxon change status from draft to withdrawn
+      are_you_sure_you_want_to_withdraw_this_taxon_change: Are you sure you want to withdraw this taxon change?
       move_children_desc: |
         Children of the input taxon that are genera or coarser will be moved to
         the output; species, subspecies, varieties, etc. will be swapped into
         equivalents with the new genus or species name.
+      # act of changing a taxon change status from withdrawn back to draft meaning reinstated as a draft
+      restore: restore
+      # in process of changing a taxon change status from withdrawn back to draft meaning reinstated as a draft
+      restoring: restoring
+      # act of changing a taxon change status from draft to withdrawn meaning abandoned
+      withdraw: withdraw
```

Comment:
This text is also used in a button and should be capitalized

Reply:
capitalized